packages/ace-report-axe/src/index.js

```diff
@@ -105,18 +105,21 @@ function  axe2ace(spineItem, axeResults, lang) {
 
         // https://github.com/dequelabs/axe-core/blob/v3.2.2/lib/core/reporters/helpers/failure-summary.js
         // // https://github.com/dequelabs/axe-core/blob/v3.2.2/lib/misc/none-failure-summary.json#L4
-        // description = description.replace("Fix all of the following:", "");
+        // Fix all of the following:
         // // https://github.com/dequelabs/axe-core/blob/v3.2.2/locales/fr.json#L664
-        // description = description.replace("Corriger tous les éléments suivants :", "");
+        // Corriger tous les éléments suivants :
         // // https://github.com/dequelabs/axe-core/blob/v3.2.2/lib/misc/any-failure-summary.json#L4
-        // description = description.replace("Fix any of the following:", "");
+        // Fix any of the following:
         // // https://github.com/dequelabs/axe-core/blob/v3.2.2/locales/fr.json#L656
-        // description = description.replace("Corriger l’un des éléments suivants :", "");
-
-        // regexp for the above
-        description = description.replace(/[^:]+:/, "");
+        // Corriger l’un des éléments suivants :
 
+        // TODO: this is a hacky way to fix the Axe bug that forces the wrong language (en) into `failureSummary`.
+        description = description.replace(/Fix any of the following:/g, "");
+        description = description.replace(/Fix all of the following:/g, "");
         description = description.trim();
+        description = description.replace(/[\n]+/g, "\n");
+        description = description.replace(/\n/g, " --- ");
+        description = description.replace(/\s+/g, " ");
```

Comment:
@danielweck 

I've tested it here, and it works. I just have a suggestion: if you remove the last three lines of regex, then you retain the former design of the report (bullets), which looked better, in my opinion.

Reply:
Yes, you are correct, the HTML report looks bad. I fixed the normalization function for the `dct:description` JSON field, but I must also ensure the contents are rendered well in the HTML markup. I will be looking into this shortly.

Reply:
Actually this was a "typo" on my part, thank you for catching this! Correcting now.

Reply:
Done: https://github.com/daisy/ace/commit/6f624bcef18e59fffe46ec63f658b61e37e6435d
Sorry about that, the collapsing of whitespaces included line breaks, and I forgot to re-add them as separators. Thank you for checking, much appreciated!